style.css

```diff
@@ -48,11 +48,62 @@ body {
   }
 }
 
-.material-icon {
+.menu-icon {
   width: 24px;
   height: auto;
 }
 
+.hamburger {
+  z-index: 100;
+  top: 1rem;
+  right: 1rem;
+  padding: 0;
+  border: #fff;
+
+  /* background: white; */
+  cursor: pointer;
+  color: #6070ff;
+}
+
+.close-icon {
+  display: none;
+  background: #6070ff;
+  color: #fff;
+}
+
+.menu {
+  position: absolute;
+  transform: translateY(-100%);
+  transition: transform 0.2s;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #6070ff;
+  opacity: 0.9;
+  color: white;
+  list-style: none;
+  padding-top: 4rem;
+  padding-left: 2rem;
+  gap: 12px;
+}
```

Comment:
Please notice that, in the original design, the background is blurred when the mobile menu appears 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/201729027-17c6b6eb-7bcd-4c7a-bd14-105c1be3d8cf.png)  | ![image](https://user-images.githubusercontent.com/42799579/201729003-d06c5106-8f6a-4705-bafc-c5e33efd6f99.png)  |

Reply:
Issue resolved. Opacity has been increased to blur the background since background could not be blurred directly